libs/common/src/test-helpers/index.ts

```diff
@@ -1 +1,3 @@
 export * from './mock-data';
+// './test-cases' uses requires mongo related functionality, so should be in a separate export
+// export * from './test-cases';
```

Comment:
Should this be commented out?

Reply:
Yes - same idea as `libs/common/src/index.ts` where the `export * from './sql-db-connection'` is commented out

Reply:
(sorry again, not looking at the file rn, but adding a comment to be annoying and document if anyone looks at this review in future) It's doing this to basically say "deliberate omission"? Happy with this -- I'd like the comment to quicklty touch on why mongo functionality is a no go with general exports